Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -576,6 +576,26 @@ define([
          */
         this.showMemoryUsage = false;
 
+        var showUrl = knockout.observable();
+        knockout.defineProperty(this, 'showUrl', {
+            get : function() {
+                return showUrl();
+            },
+            set : function(value) {
+                showUrl(value);
+                if (defined(that._tileset)) {
+                    that._tileset.debugShowUrl = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to show tile url.  This property is observable.
+         *
```

Comment:
We only need `@memberof` when the property is added using `defineProperties`.  So it can probably be removed from the doc of all of these since they're directly on `this`